src/pages/Product/Product.js

```diff
@@ -1,9 +1,178 @@
 import React, { Component } from 'react';
 
+// import { GET_PRODUCTS_API } from '../../../src/config.js';
+
+import ProductCard from './ProductCard/ProductCard';
+
+import './Product.scss';
+
 export class Product extends Component {
+  constructor() {
+    super();
+    this.state = {
+      productCard: [],
+      addCart: false,
+      bone: false,
+      hair: false,
+      growth: false,
+      skin: false,
+    };
+  }
+
+  componentDidMount() {
+    // fetch(`${GET_PRODUCTS_API}`)
+    fetch('./data/ProductData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productCard: data.message,
+        });
+      });
+  }
+
+  makeCondition = () => {
+    let fillter = [];
+
+    if (this.state.bone) {
+      fillter.push(`efficacy=1`);
+    }
+
+    if (this.state.hair) {
+      fillter.push(`efficacy=2`);
+    }
+
+    if (this.state.growth) {
+      fillter.push(`efficacy=3`);
+    }
+
+    if (this.state.skin) {
+      fillter.push(`efficacy=4`);
+    }
+
+    // fetch(`${GET_PRODUCTS_API}?`)
+    fetch(`./data/ProductData.json?${fillter.join('&')}`)
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productCard: data.message,
+        });
+      });
+  };
+
+  handleCheckBox = event => {
+    this.setState(
+      {
+        [event.target.name]: !this.state[event.target.name],
+      },
+      () => {
+        this.makeCondition();
+      }
+    );
+    console.log(this.state);
+  };
+
   render() {
-    return <div>Product</div>;
+    const { productCard } = this.state;
+
+    return (
+      <div className="Product">
+        <header className="productHeader">
+          <h1>
+            건강한 삶을 위한
+            <br />
+            빌리의 연구와 도전은 계속됩니다.
+          </h1>
+        </header>
+        <section className="productBody">
+          <h2 className="sr-only">Product Body</h2>
+
+          <form className="productCategory">
+            <div className="ProductCheckBox">
+              <input
+                id="bone"
+                type="checkbox"
+                name="bone"
+                onChange={this.handleCheckBox}
+              />
+              <label for="bone">
+                <img className="iconImage" alt="hair" src="images/bone.svg" />
+                <span className="categoryText">뼈</span>
+              </label>
+            </div>
+            <div className="ProductCheckBox">
+              <input
+                id="hair"
+                type="checkbox"
+                name="hair"
+                onChange={this.handleCheckBox}
+              />
+              <label for="hair">
+                <img
+                  className="iconImage"
+                  alt="hair"
+                  src="images/hairstyle.svg"
```

Comment:
```suggestion
                  src="/images/hairstyle.svg"
```
수정님 images/hairStyle.svg로 하시게되면 서버의 root 폴더가 아니라 현재 url위치에서 상대적으로 경로를 찾게됩니다!
즉 만약에 localhost:3000/detail 에서 위 페이지를 불러온다고 했을 때
public/images/hairstyle.svg를 찾는게 아니라 현재위치에서 상대적으로 public/detail/images/hairStyle.svg로 찾게됩니다!
그러면 당연히 이미지가 정상적으로 불러와지지 않겠죠?
다른 이미지들도 마찬가지로 수정해주세요!

Reply:
오! 그렇네요 어쩐지 merge전에는 이미지가 잘 보였는데 그 후(?)에는 안보이는 상황이 있었어요!! 감사합니다!!